compiler/codegen/cg-symbol.cpp

```diff
@@ -1723,6 +1724,28 @@ void TypeSymbol::codegenAggMetadata() {
 #endif
 }
 
+#ifdef HAVE_LLVM
+#if 0
```

Comment:
Why is this ifdef'd out? It can be here not ifdefed out and just never used, right?

Reply:
I wanted to make sure I didn't trip any defined-but-not-used warnings since it's not used yet here.